docs/fast_data/runtime_management/database_setup.mdx

```diff
@@ -0,0 +1,100 @@
+---
+id: cp_database_setup
+title: Database Preparation
+sidebar_label: Database Preparation
+---
+
+:::info
+Currently only MongoDB is supported as underlying database for storing the application data.
+:::
+
+:::caution
+It is highly recommended to employ a **dedicated** database / namespace to store Fast Data Runtime Management data. This allows
+restricting which applications have access to the management collections and permits to keep the necessary collections
+all under the same location without additional clutter from other projects.
+:::
+
+Fast Data Runtime Management solution relies on a set of collections to carry out most of its tasks, that is
+keeping track of runtime views or updating the desired Fast Data Pipelines state. In order to provide
+a performant and reliable system, the database must be configured accordingly to support application execution.
+
+This means that required collections must be created, necessary index added and additional schema validation set on them,
+so that only valid data can be read and written to those collections.  
+
+To reach this objective in a straightforward manner we provide an administration command-line interface,
+which takes care of creating all the necessary resources on selected database. This CLI is offered
+within as a Docker image, which can be run within your namespace every time a new application version is released.
+
+## CLI Configuration
+
+Running the CLI inside your project can be done exploiting a K8s cronjob. In this [documentation page](/development_suite/api-console/api-design/jobs-cronjob.md) it is described how it can be configured, while here
+are highlighted which specific configurations of the K8s manifest:
+
+- `<URL_TO_CONTAINER_REGISTRY>` &rarr; is the url to your Container Registry of reference;
+- `data-fabric/fabric-admin` &rarr; image is employed, which should be found within your Container Registry;
+- `<TAG>` &rarr; is the latest stable version of CLI image;
+- `suspend` &rarr; the cronjob must be released as suspended, since it should be launched manually only once;
+- `<NAME_CONTAINER_REGISTRY_SECRET>` &rarr; name of the secret where are contained the credentials for connecting to your Container Registry;
+- `MONGODB_ADMIN_URL` &rarr; connection string to your MongoDB database.
```

Comment:
I'd put this under the example so the user knows where each of these are used in the manifest